### ISSUE TITLE
Why is the current NPM release `1.0.0` different?

### ISSUE THREAD

**Issue by nicholaschiang** [sidorares/json-bigint#74]
There are significant differences between the `v1.0.0` tag and the `master` branch. Why? Why haven't you released a new version to NPM? The newest version there (`v1.0.0`) is over two years old...

**Comment by nicholaschiang** [comment]
@sidorares there seems to have been significant changes and bug fixes since that `v1.0.0` tag. Could you push those to NPM?

**Comment by sidorares** [comment]
yes I probably should. Due to a significant time gap between releases major version bump might be a good idea ( not that I'm aware of any backwards incompatibilities, just as a precaution )

**Comment by nicholaschiang** [comment]
Would you need help with this? I'd be more than happy to contribute!

**Comment by sidorares** [comment]
> Would you need help with this? I'd be more than happy to contribute!

thanks for the offer! I'm slowly migrating all the projects to gh actions - if you have time to help with that would be great

**Comment by tomercohen1210** [comment]
@sidorares do you think the new version will come out soon?

**Comment by ldt1997** [comment]
I meet problems with float numbers and would like to use the new version

**Comment by justinpark** [comment]
@sidorares any updates for the release? I also encounter the same problems with float numbers and would like to use the new version

**Comment by ThallesP** [comment]
Hi @sidorares could you at least publish the latest version on NPM? Even doing manually would be good with `npm publish`.

Currently I've to do an NPM install from this Github repository, locked at a specific commit, which isn't that good.

**Comment by sidorares** [comment]
@ThallesP I promise I'll give this library some attention over a coming weekend.

**Comment by aladdin-add** [comment]
@sidorares friendly ping! :)

**Comment by riflowth** [comment]
@sidorares Friendly reminder. I face some problem in my production code (related to https://github.com/sidorares/json-bigint/issues/49)

**Comment by nathanmmiller** [comment]
@sidorares should we expect a release with the last four years' worth of code changes or is the solution to use a forked package?

**Comment by bhufmann** [comment]
@sidorares could you please release a latest version to NPM? In advance thank you very much.

**Comment by Suntgr** [comment]
@sidorares Is there any issue with publishing the latest code to npm? I'm more than happy to help.

**Comment by bagustambunan** [comment]
Hi sir @sidorares , could you please publish the latest version to NPM?

**Comment by ThallesP** [comment]
If anyone is looking for a well maintained alternative, we've been using https://www.npmjs.com/package/lossless-json for quite sometime now

**Comment by Ivan-Korolenko** [comment]
I would also like to add an alternative library here. [json-with-bigint](https://www.npmjs.com/package/json-with-bigint)

Its main feature is the ability to do consistent round-trip operations with JSON, but it also provides a lot of other DX-oriented improvements regarding the problems it solves, like support of TS, float and negative numbers, pretty printed JSON, etc.

If you're looking for an actively maintained library that handles BigInt and JSON, check it out.
